src/ocm/components/clusterConfiguration/manifestsConfiguration/components/FolderDropdown.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import {
+  DropdownItem,
+  DropdownToggle,
+  Dropdown,
+  HelperText,
+  FormGroup,
+} from '@patternfly/react-core';
+import { CaretDownIcon } from '@patternfly/react-icons';
+import { useField } from 'formik';
+import { getFieldId } from '../../../../../common/components/ui/formik';
+import { PopoverIcon } from '../../../../../common';
+
+const FolderLabel = () => {
+  return (
+    <>
+      Folder{' '}
+      <PopoverIcon
+        bodyContent={
+          <>
+            Use documentation to help you select the right folder for the manifest you want to add.{' '}
```

Comment:
It would be helpful if we had a link here. WDYT?

Reply:
True. I've added the link that Avisay suggested in the ticket: https://docs.openshift.com/container-platform/4.12/installing/install_config/installing-customizing.html